### ISSUE TITLE
conda install: requirements incompatible for 3+

### ISSUE THREAD

**Issue by endrebak** [jdidion/atropos#11]
```
conda install atropos
Fetching package metadata ...............
Solving package specifications: .


UnsatisfiableError: The following specifications were found to be in conflict:
  - atropos -> khmer -> python 2.7*
  - python 3.6*
Use "conda info <package>" to see the dependencies for each package.
```

**Comment by jdidion** [comment]
Thanks for the report. Working on this now.

**Comment by jdidion** [comment]
Apologies - I failed to update the MANIFEST.in after moving some code around. It is fixed now in v1.1.2 and I have verified that it works in py36 via pip. Still verifying that it works via conda.
